src/main/java/org/sagebionetworks/bridge/hibernate/HibernateAccountDao.java

```diff
@@ -86,6 +88,18 @@ protected String generateGUID() {
         return BridgeUtils.generateGuid();
     }
     
+    @Override
+    public List<String> getStudyIdsForUser(String synapseUserId) {
+        if (isBlank(synapseUserId)) {
+            return ImmutableList.of();
+        }
```

Comment:
Because of the way the Account classes evolved, we have no AccountService. So the AccountDao ends up doing some of the things a service class would do. It's on my long list of things to refactor. I could check this in the two controllers that call it, but then it would be duplicated in both controllers.